modules/battery_manager/ajax/add_entry.php

```diff
@@ -44,7 +46,7 @@ function checkForDuplicate()
         if ($i > 0) {
             $connector = "AND ";
         }
-        if (isset($value)) {
+        if (isset($value) || $value !== null) {
```

Comment:
@victoriafoing I think @ridz1208 wanted you to check for an empty string `""` here instead of null.

Reply:
```php
if (isset($value) && !empty($value)) {
```